wagtail/snippets/templates/wagtailsnippets/snippets/edit.html

```diff
@@ -62,6 +62,38 @@
             });
 
             LockUnlockAction('{{ csrf_token|escapejs }}', '{{ action_url }}');
+            ActivateWorkflowActionsForEditView('[data-edit-form]');
+
+            {% if publishing_will_cancel_workflow %}
+                /* Make user confirm before publishing the snippet if it will cancel an ongoing workflow */
```

Comment:
Really wish I didn't have to add more to the inline scripts, but I'm just copying this from pages and I think we can refactor them both together later, possibly with Stimulus